tests/test_api.py

```diff
@@ -508,6 +508,7 @@ def test_pop():
         ("https://foo/", b"foo=bar", "https://foo/", b"foo=bar"),
         ("https://foo/", [("foo", "bar")], "https://foo/", [("foo", "bar")]),
         ("https://foo/", {"foo": "bar"}, "https://foo/", {"foo": "bar"}),
+        ("https://foo/", ("foo", "bar"), "https://foo/", ("foo", "bar")),
```

Comment:
This should probably be a tuple of tuple, if the intention is to test `?foo=bar`?

```suggestion
        ("https://foo/", (("foo", "bar"),), "https://foo/", (("foo", "bar"),)),
```

Reply:
Yes you're right. Oopsie